Cargo.toml

```diff
@@ -18,7 +18,6 @@ members = [
   "vm/sdk",
 ]
 exclude = ["examples"]
-resolver = "2"
```

Comment:
Was this intentional? I'm getting warnings:

```
warning: virtual workspace defaulting to `resolver = "1"` despite one or more workspace members being on edition 2021 which implies `resolver = "2"`
note: to keep the current resolver, specify `workspace.resolver = "1"` in the workspace root's manifest
note: to use the edition 2021 resolver, specify `workspace.resolver = "2"` in the workspace root's manifest
note: for more details see https://doc.rust-lang.org/cargo/reference/resolver.html#resolver-versions
```

Reply:
I don't understand how it works. I thought that "2" was the default, but apparently, it is not. I reverted, and I will come back to it later.